src/mappings/balancerLiquidityMiningMapping.ts

```diff
@@ -23,15 +26,15 @@ export function handleRewardAdded(event: RewardAdded): void {
 }
 
 export function handleRewardPaid(event: RewardPaid): void {
-  onRewardPaid(event.address, event.transaction.hash.toHex(), event.params.user.toHex(), BALANCER_LM);
+  onRewardPaid(
+    new OnRewardAddedParams(event.address, BALANCER_LM, event.params.user.toHex(), event.transaction.hash.toHex())
```

Comment:
@mohammadranjbarz  Params are passed in wrong order, check all other places like this.

Reply:
I did a mistake, it was fine.